.github/workflows/ci.yaml

```diff
@@ -22,12 +22,18 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
+    - name: setup-docker
```

Comment:
AFAIK docker is already setup on runners, so why do we need it ?

Reply:
I didn't know that it could be used as a command line. I'll try without it.

Reply:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
here are all packages installed. There is also `oc` there :-)

Reply:
removed. Thanks for the information !